marlowe-dashboard-client/src/Component/Contacts/State.purs

```diff
@@ -188,11 +201,24 @@ toWalletIdInput ::
 toWalletIdInput = mapSubmodule _walletIdInput WalletIdInputAction
 
 ------------------------------------------------------------
+-- The cardano Blockchain has Multi-Asset support, each monetary policy is identified
+-- by a policyId which is also known as the CurrencySymbol. The ADA token is a special
+-- case that has the empty string as both the token name and the currency symbol
+-- https://docs.cardano.org/native-tokens/learn
+-- https://github.com/input-output-hk/plutus/blob/1f31e640e8a258185db01fa899da63f9018c0e85/plutus-ledger-api/src/Plutus/V1/Ledger/Ada.hs#L45
```

Comment:
There is already: https://github.com/input-output-hk/purescript-web-common/tree/main/src/Plutus/V1/Ledger

Reply:
Cool, we can put this next to that